CONTRIBUTING.md

```diff
@@ -1,9 +1,15 @@
-# Contributing to QUIC
+# Contributing to QUIC base-drafts
 
-Anyone can contribute to QUIC; you don't have to join the Working Group, because there is no "membership" -- anyone who participates in the work, as outlined below, is part of the QUIC Working Group.
+The base-drafts repository is the home of the following QUIC Working Group documents:
 
-Before doing so, it's a good idea to familiarize yourself with our current [issues list](https://github.com/quicwg/base-drafts/issues) and [charter](https://datatracker.ietf.org/wg/quic/about/). If you're
-new to this, you may also want to read the [Tao of the IETF](https://www.ietf.org/tao.html).
+* Invariants
+* Transport
+* TLS
+* HTTP/3
+* QPACK
+* Recovery
+
+**All of the documents have now passed IESG review stage. We will no longer consider Design changes or substantial Editorial changes unless they relate to severe security, interoperabily or deployment problems. See [Post-IESG Process](#post-iesg-process) below for further information** 
```

Comment:
Maybe instead:

> All of the documents have been approved for publication as RFC. ...

That might be more accessible to people who aren't familiar with our processes.